test/runtests.jl

```diff
@@ -1,2 +1,6 @@
-# write your own tests here
-include("singlethreaded.jl")
+using ReadWriteLocks
+using Test
+
+@testset "ReadWriteLocks" begin
+    include("singlethreaded.jl")
```

Comment:
What about the `"Two-threaded tests"` testset?

Reply:
you named it 😆 I will just move them into the runtests file 😄

Reply:
Oh lol, maybe because they were never actually run on multiple threads?